platforms/arm/stm32/fastpin_arm_stm32.h

```diff
@@ -96,7 +102,58 @@ _DEFPIN_ARM(19, 2, A);
 
 #define HAS_HARDWARE_PIN_SUPPORT
 
-#endif
+#endif // SPARK
+
+#if defined(__STM32F1__)
+
+_IO32(A); _IO32(B); _IO32(C);
+
+#define MAX_PIN PB1 // FIXME need to be a number?
```

Comment:
Really all the defpin values should be numbers - 0-15.  Please don't use the lettered pin numbers, as many platforms/environments do weird things with remapping them (and in at least one case I saw, they were macros).

There's a reason why all the FastLED demos refer to pins as 4, or 12 vs. D4 or D12.  This will 'work' - mostly - but it also breaks the convention in the library, and I'd like to not toss more confusion into the mix on how pins should be referenced.

Reply:
Done (except the range is 10-46 as per pin definitions at say https://wiki.stm32duino.com/index.php?title=File:Bluepillpinout.gif)